1.1 Desktop app calls Web API - PoP/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,99 @@
+{
+  "Sample": {
+    "Title": "A WPF application that calls a Web API running on ASP.NET Core 2.0 protected by Azure AD using OAuth 2.0 access tokens. ",
```

Comment:
It's a console app now no?

Reply:
Also the WPF screenshots are no longer relevant.

Reply:
I could re-introduce the WPF application now. Wondering about that.
Right the screenshots need to be updated